tests/test_tmaze.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+
+from grl.mdp import MDP, AbstractMDP
+from grl.environment.tmaze_lib import tmaze
+
+
+if __name__ == "__main__":
+    corridor_length = 0
+    T, R, gamma, p0, phi = tmaze(corridor_length)
+    mdp = MDP(T, R, p0, gamma=gamma)
+    pomdp = AbstractMDP(mdp, phi)
+
+    n_samples = 10000
+
+    # first we test our start states
```

Comment:
Can you make the tests in both files like the ones in e.g. `test_policy_eval.py`? That way they run and show up nicely in pytest (and can be separated out too if it seems helpful)

Reply:
If you have tests that need to be configured in the same way, you can use [pytest fixtures](https://docs.pytest.org/en/6.2.x/fixture.html#what-fixtures-are) to accomplish that.

Here's an example of how that works:
https://github.com/camall3n/visgrid/blob/master/test/test_gridworld.py